@here/harp-omv-datasource/lib/IGeometryProcessor.ts

```diff
@@ -40,7 +24,7 @@ export interface ILineGeometry {
     /**
      * The positions of this [[IlineGeometry]] projected in world space.
```

Comment:
Same as above, please fix comment to mention the correct space

Reply:
Done.